docs/index.md

```diff
@@ -6,328 +6,609 @@ description: Get started with creating Flox environments.
+Next you'll install your dependencies via [`flox install`][install].
+This transactionally builds your environment with the new packages so that
+it's never left in a broken state.
+The packages in your environment aren't available until you activate the
+environment,
+which we'll do in the next step,
+but this means that the packages in a Flox environment don't get in your way
+when you aren't using an environment.
 
 ```
-flox_project $ which httpie
-httpie not found
+my_project $ flox install go
 ```
 
-The packages in a Flox environment don't pollute your system when you aren't
-using them.
+You can also install specific versions at the command line using semver syntax:
 
-## Sharing the environment
+```
+my_project $ flox install go@1.22.6
+```
 
-One of the most powerful features of Flox is the ability to share environments.
-Let's push our new environment to FloxHub so that we can share it with other
-people.
+## Activate the environment
 
-We'll run the [`flox push`][push] command to push our environment to FloxHub.
-The first thing this will ask you to do is authenticate with FloxHub by entering
-a code that it displays in the terminal.
+In order to use the packages in the environment you need to _activate_ it
+via [`flox activate`][activate].
+This puts you into a subshell with various environment variables set
+e.g. `PATH`, `CPATH`, `PKG_CONFIG_PATH`, `PYTHONPATH`, etc in such a way that
+all of the packages in your environment should be available.
 
-```
-flox_project $ flox push
-You are not logged in to FloxHub. Logging in...
-> First copy your one-time code: <code>
+For instance, if you installed `go` 1.22.6 above,
+activating your environment would look something like this:
 
-Press enter to open hub.flox.dev in your browser...
 ```
+my_project $ go version
+fish: Unknown command: go
 
-Copy the code and press Enter.
-This will open your browser,
-where you can enter the code and log in to FloxHub.
-This will use GitHub to establish your identity,
-but it won't grant Flox any permissions to read or write to any of your
-repositories.
-Once this is done you can return to your terminal and you should see the
-confirmation message, where `<user>` is your GitHub username:
+my_project $ flox activate
+✅ You are now using the environment 'my_project'.
+To stop using this environment, type 'exit'
 
+flox [my_project] my_project $ go version
+go version go1.22.6 darwin/arm64
 ```
-✅ Authentication complete
-✅ Logged in as <user>
-✅ flox_project successfully pushed to FloxHub
 
-Use 'flox pull <user>/flox_project' to get this environment in any other location.
+When you activate the environment you can see that `flox [my_project]` is
+prepended to your prompt so that you can see that you're inside the
+environment.
+If you have more than one environment activate (you can stack them!) this prompt
+will show a list of the active environments.
+Whether to show this prompt is configurable, see [`flox config`][config]
```

Comment:
This feels like a good time to mention what to do if you don't see the prompt (bc of zsh theme, etc). Perhaps?

> If you don't see the prompt change, <do X>. If you want to disable the prompt, see  [`flox config'][config].

If there isn't a simple thing they can do, can we just say "If you use custom zsh themes, your prompt may not change."